command/job/hunt.go

```diff
@@ -0,0 +1,182 @@
+package job
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+)
+
+func Hunt() error {
+	// create Nomad API client
+	nomadClient, err := api.NewClient(api.DefaultConfig())
+	if err != nil {
+		return err
+	}
+
+	// Get the jobs
+	jobs, _, err := nomadClient.Jobs().List(nil)
+	if err != nil {
+		return err
+	}
+
+	for _, job := range jobs {
+		if job.Type != "service" {
+			continue
+		}
+
+		// Get job's running allocations
+		jobAllocations, _, err := nomadClient.Jobs().Allocations(job.ID, true, nil)
+		if err != nil {
+			return err
+		}
+
+		var runningAllocationJobVersion []uint64
+		for _, allocation := range jobAllocations {
+			if allocation.ClientStatus == "running" {
```

Comment:
if you want to print all allocations anyways, then check for running and unmatched job version and print here

Reply:
I think I understood what you meant.
I reworked the code, it's one less for loop, but from my point of view, it's less readable or something like this.
please take a look